pelicanconf.py

```diff
@@ -48,7 +49,7 @@
 
 #### SPONSORS ####
 SPONSORS = (('Eclipse Foundation', 'https://www.eclipse.org', 'images/eclipse-foundation.svg'),
-            ('ADLINK Technology', 'https://www.adlinktech.com', 'images/adlink.png'),)
+            ('ADLINK Technology', 'https://www.adlinktech.com', 'images/company_logo.svg'),)
```

Comment:
Better name this `adlink.svg` since other companies might start contributing as well. e.g. APEX.AI might want it's logo here.